csharp/src/Google.Protobuf.Test/Reflection/CustomOptionsTest.cs

```diff
@@ -99,66 +99,6 @@ public class CustomOptionsTest
     {
         delegate bool OptionFetcher<T>(int field, out T value);
 
-        [Test]
```

Comment:
They were deleted since they tested a part of how CustomOptions were implemented before. Since those implementation details don't apply anymore, the tests should be deleted. Specifically:

* EmptyOptionsIsShared tested to make sure two empty CustomOptions instances are the same empty instance

* SimpleIntegerTest tested parsing fields, something CustomOptions no longer does

Reply:
Since they are only test internal implementations, they are safe to be deleted. Thanks